vggish/vggish_input.py

```diff
@@ -20,6 +20,10 @@
 
 import resampy
 
+import sys
+import os
+sys.path.append(os.getcwd())
```

Comment:
is this needed?

Reply:
Yes because py_modules seem to be imported on order ` vggish/vggish_input.py` requires to be able to see `vggish`

Reply:
maybe the order in `py_modules` is wrong then